workflows_api/runtime/src/collection_publisher.py

```diff
@@ -183,20 +181,4 @@ def generate_stac(
         create_function = self.func_map.get(data_type, self.create_cog_collection)
         return create_function(dataset)
 
-    def ingest(self, collection: DashboardCollection, token: str, ingest_api: str):
-        """
-        Takes a collection model,
-        does necessary preprocessing,
-        and loads into the PgSTAC collection table
-        """
-        
-        url = f"{ingest_api}/collections"
-        headers = {
-            "Authorization": f"Bearer {token}",
-            "Content-Type": "application/json",
-        }
-        response = requests.post(url, data=collection.json(by_alias=True), headers=headers)
-        if response.status_code == 200:
-            print("Success:", response.json())
-        else:
-            print("Error:", response.status_code, response.text)
+    
```

Comment:
There were apparently two `ingest` methods defined - that tripped me up for a while 😓

Reply:
oh no! 🙃